packages/http-client-csharp/generator/Microsoft.Generator.CSharp/src/PostProcessing/GeneratedCodeWorkspace.cs

```diff
@@ -52,9 +51,9 @@ public void AddPlainFiles(string name, string content)
             PlainFiles.Add(name, content);
         }
 
-        public async Task AddGeneratedFile(string name, string text)
+        public async Task AddGeneratedFile(CodeFile codefile)
```

Comment:
well we put `CodeFile` here therefore codeFile has to be public.
But it feels like this is a thing that only used in our core part of the generator, which would be better if `CodeFile` is internal.
But I did not see the other way around